packages/usa-date-picker/src/styles/_usa-date-picker.scss

```diff
@@ -458,7 +458,7 @@ $navigate_far_next: map-merge(
     background-image: none;
 
     &:hover {
-      outline: 2px solid transparent;
+      border: $border-high-contrast;
```

Comment:
I am not seeing previous-year-chunk. Can you show me what this does?

Reply:
Previous and next year chunk appears when you open the year select page. 461 adds this blue outline on hover:

![image](https://user-images.githubusercontent.com/25211650/226926610-8daccee6-2669-42ca-bd4c-530e3b0cb620.png)

Reply:
Thanks for that. I see it now :) What do you think about adding this hover state to the next/previous month/year arrows as well? 

![image](https://user-images.githubusercontent.com/93996430/227370379-483861cd-82a9-430d-be51-0976c1cc031b.png)

Reply:
I'm unable to add a border to these because it is hidden by the icon-mask. I tried adding a border to the parent div but it caused a content shift when the border would pop in and out. This is the same reason the calendar icon changes color on hover instead of adding a border!

I was able to change the background color of the icons instead to match the calendar icon:

![image](https://user-images.githubusercontent.com/25211650/227562182-90724d0c-5e13-4cb2-9e5c-cfbff1b4612d.png)

Do you think I should do the same for the next year chunks instead of the border I added?
![image](https://user-images.githubusercontent.com/25211650/227562376-ededf2f2-d98f-476b-b98d-95e38a0c101d.png)

Reply:
What do you think about using an outline with a negative outline-offset on the parent? I consider you the expert on the colors and border widths for HC mode so I might get the specifics wrong here, but here is a sample implementation:

```
.usa-date-picker__calendar__cell--center-items:hover {
    outline: 2px solid activeText; 
    outline-offset: -2px; 
}
```

Reply:
@amyleadem That worked like a charm! Great idea

Resolved in [bd7de74](https://github.com/uswds/uswds/pull/5147/commits/bd7de74af2a03685f6d4b870a86f2caaa87bae51)